client/gutenberg/extensions/contact-form/components/jetpack-contact-form.jsx

```diff
@@ -11,6 +11,71 @@ import { Component, Fragment } from '@wordpress/element';
  * Internal dependencies
  */
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+const ALLOWED_BLOCKS = [
+	'jetpack/markdown',
+	'core/paragraph',
+	'core/image',
+	'core/heading',
+	'core/gallery',
+	'core/list',
+	'core/quote',
+	'core/shortcode',
+	'core/archives',
+	'core/audio',
+	'core/categories',
+	'core/code',
+	'core/cover',
+	'core/embed',
+	'core-embed/twitter',
+	'core-embed/youtube',
+	'core-embed/facebook',
+	'core-embed/instagram',
+	'core-embed/wordpress',
+	'core-embed/soundcloud',
+	'core-embed/spotify',
+	'core-embed/flickr',
+	'core-embed/vimeo',
+	'core-embed/animoto',
+	'core-embed/cloudup',
+	'core-embed/collegehumor',
+	'core-embed/dailymotion',
+	'core-embed/funnyordie',
+	'core-embed/hulu',
+	'core-embed/imgur',
+	'core-embed/issuu',
+	'core-embed/kickstarter',
+	'core-embed/meetup-com',
+	'core-embed/mixcloud',
+	'core-embed/photobucket',
+	'core-embed/polldaddy',
+	'core-embed/reddit',
+	'core-embed/reverbnation',
+	'core-embed/screencast',
+	'core-embed/scribd',
+	'core-embed/slideshare',
+	'core-embed/smugmug',
+	'core-embed/speaker',
+	'core-embed/speaker-deck',
+	'core-embed/ted',
+	'core-embed/tumblr',
+	'core-embed/videopress',
+	'core-embed/wordpress-tv',
+	'core/file',
+	'core/html',
+	'core/media-text',
+	'core/latest-comments',
+	'core/latest-posts',
+	'core/preformatted',
+	'core/pullquote',
+	'core/separator',
+	'core/block',
+	'core/spacer',
+	'core/subhead',
+	'core/table',
+	'core/template',
+	'core/verse',
+	'core/video',
+];
```

Comment:
Not a big fan of the whitelist approach. But it works as described!

A black list approach would look like:

```
import { getBlockTypes } from '@wordpress/blocks';

const DISALLOWED_BLOCKS = [
	'jetpack/contact-form'
];

const allowedBlocks = getBlockTypes().map( block => block.name ).filter( name => ! DISALLOWED_BLOCKS.includes( name ) );
```

Reply:
For so many items, whitelist doesn't make a lot of sense to me — I'd assume you want to keep it up to date with new blocks that come in core.

If the idea is to just add paragraph, image, and maybe one or two others, whitelist seems fine.